lib/sql.js

```diff
@@ -1007,8 +1007,23 @@ Connector.defineAliases(SQLConnector.prototype, 'replace', ['replaceAll']);
  * @param {object} where An object for the where conditions
  * @returns {ParameterizedSQL} The SQL WHERE clause
  */
-SQLConnector.prototype.buildWhere = function(model, where) {
-  const whereClause = this._buildWhere(model, where);
+SQLConnector.prototype.buildWhere = function(model, where, options) {
+  let relationType = '';
+  let relationKeyFrom = '';
+  if (options && options['model'] && options['model']['definition']) {
```

Comment:
Ideally this should be part of datasource juggler repo as thats where we implement relations.

Reply:
Hi, @samarpanB. The relations are defined in the datasource-juggler, but here I am trying to modify the SQL queries to enable the `inq` filter for referencesMany relations.